pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -155,18 +156,16 @@ func (e *Streamer) doStream() error {
 	}
 	defer conn.Close()
 
-	doneChan := make(chan struct{}, 2)
 	errorChan := make(chan error)
+	var wg sync.WaitGroup
 
 	cp := func(s string, dst io.Writer, src io.Reader) {
+		defer wg.Done()
```

Comment:
The unit tests mock reality... @ncdc can probably give more details about the behavior of stdin streams in real life